tests/benchmarks/module/module_benchmark_test.py

```diff
@@ -0,0 +1,62 @@
+# stdlib
+from typing import Any
+from typing import Callable
+from typing import List
+
+# third party
+import numpy as np
+import syft as sy
+import torch
+
+from sympc.session import Session
+from sympc.session import SessionManager
+from sympc.tensor import MPCTensor
+
+
+class ConvNet(sy.Module):
+    def __init__(self, torch_ref):
+        super(ConvNet, self).__init__(torch_ref=torch_ref)
+        self.conv1 = self.torch_ref.nn.Conv2d(
+            in_channels=1, out_channels=5, kernel_size=5
+        )
+        self.fc1 = self.torch_ref.nn.Linear(2880, 10)
+        self.fc2 = self.torch_ref.nn.Linear(10, 5)
+
+    def forward(self, x):
+        x = self.conv1(x)
+        x = self.torch_ref.nn.functional.relu(x)
+        x = x.view(1, -1)
+        x = self.fc1(x)
+        x = self.fc2(x)
+        x = self.torch_ref.nn.functional.relu(x)
+        return x
+
+
+def run_conv_model(get_clients: Callable[[int], List[Any]]):
+    model = ConvNet(torch)
+
+    clients = get_clients(2)
+
+    session = Session(parties=clients)
+    SessionManager.setup_mpc(session)
+
+    mpc_model = model.share(session=session)
+
+    x_secret = torch.randn((1, 1, 28, 28))
+    x_mpc = MPCTensor(secret=x_secret, session=session)
+
+    model.eval()
+
+    # For the moment we have only inference
+    expected = model(x_secret)
+
+    res_mpc = mpc_model(x_mpc)
+    assert isinstance(res_mpc, MPCTensor)
+
+    res = res_mpc.reconstruct()
+    expected = expected.detach().numpy()
+    assert np.allclose(res, expected, atol=1e-3)
+
+
+def test_run_conv_model(benchmark, get_clients: Callable[[int], List[Any]]):
```

Comment:
**Suggestion:** can you move this function to separate file ? may be in future we will be adding more benchmarks to it.

Reply:
The run_conv_model, sure. I will try it later this afternoon.

Reply:
Done, for the moment I decided to do a single file called `conv_model`. In the future we could group different models by file: conv_models, linear_models, ... or we could use one file for each. In any case, changing the name in the future should be easy. If you are okey with the change, we can merge it!

Reply:
While doing aanuraj suggestion and fixing the errors, I decided to move benchmarks into another folder different from the tests folder to tackle gmuraru first question.
@aanurraj @gmuraru If you could do a last review before merging, would be awesome. Thanks!